samples/subsys/fs/fs_sample/boards/stm32h573i_dk.overlay

```diff
@@ -0,0 +1,11 @@
+/*
+ * Copyright (c) 2024 STMicroelectronics
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+&sdmmc1 {
+        sdmmc {
+                compatible = "zephyr,sdmmc-disk";
```

Comment:
This node shouldn't be needed, right? The `zephyr,sdmmc-disk` pulls in a driver that implements the disk API using the SDMMC subsystem. Since the ST driver implements the disk API directly without the SD subsystem, this shim layer isn't required